docker/migrations.startup

```diff
@@ -17,6 +17,7 @@ migrations() {
     python manage.py loaddata initial_category.json
     python manage.py loaddata initial_mattertype.json
     python manage.py loaddata initial_media_codes.json
+    python manage.py loaddata initial_guidance_notes.json
```

Comment:
this should only happen once otherwise the changes from the admin will be overridden.

Two options:

1. keep it as it is and remember to delete this line after the first deploy
2. delete this line and import the fixture manually when deploying the next time.

I'll leave it to you but worth noting down to delete this line if you go for option 1